<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>db36</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QTextEdit" name="log">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connect_db"/>
   <addaction name="action_sql_command"/>
   <addaction name="action_upload_blob_to_db"/>
   <addaction name="action_download_blob_from_db"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_connect_db">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res/resource.qrc">
     <normaloff>:/db_connect.png</normaloff>:/db_connect.png</iconset>
   </property>
   <property name="text">
    <string>Соединиться с БД</string>
   </property>
  </action>
  <action name="action_sql_command">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res/resource.qrc">
     <normaloff>:/command.png</normaloff>:/command.png</iconset>
   </property>
   <property name="text">
    <string>команда SQL</string>
   </property>
  </action>
  <action name="action_upload_blob_to_db">
   <property name="icon">
    <iconset resource="res/resource.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Поместить файл в БД</string>
   </property>
  </action>
  <action name="action_download_blob_from_db">
   <property name="icon">
    <iconset resource="res/resource.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Выгрузить файл из БД</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
